admin_guide/managing_networking.adoc

```diff
@@ -52,6 +52,16 @@ service named `db` in a project `myproject`, use `db.myproject.svc.cluster.local
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+To verify which networks you have joined, run:
```

Comment:
Can I suggest s/To verify which networks you have joined, run:/To verify the networks you have joined together:

I read how it is now as "I have joined on these networks" like "I have joined the meeting" and that's not the same.

Reply:
Sound good to me! Thanks, @bfallonf! :ice_cream: